quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/Environment.java

```diff
@@ -133,6 +135,10 @@ public static String getProfile() {
     public static void setProfile(String profile) {
         System.setProperty(PROFILE, profile);
         System.setProperty("quarkus.profile", profile);
+        if (isTestLaunchMode()) {
+            System.setProperty("mp.config.profile", profile);
```

Comment:
Set via maven won't work because we have programmatic invocations of this method when bootstrapping the server.

I don't think it makes sense to always set that property but keep it specific for the scenario they are related to.

Reply:
ok